librz/main/rz-sign.c

```diff
@@ -103,9 +103,6 @@ RZ_API int rz_main_rz_sign(int argc, const char **argv) {
 			break;
 		case 'v':
 			return rz_main_version_print("rz-sign");
-		case 'h':
-			rz_sign_show_help();
-			goto rz_sign_end;
 		default:
```

Comment:
I think this should remain because in the default case there is an error, in this case there isn't